client/payments/data/actions.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+import { API_NAMESPACE } from './constants';
+
+export function handleFetchError( error, message ) {
+	const { createNotice } = dispatch( 'core/notices' );
+	createNotice( 'error', message );
+
+	// eslint-disable-next-line no-console
+	console.log( error );
```

Comment:
Any reason this is left in? We don't want any console logging in production.

Reply:
@samueljseay  No particular reason actually. I copied the function from the marketing module. I will delete it.